packages/conf-libgccjit/conf-libgccjit.12/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "Lukasz Stafiniak <lukstafi@gmail.com>"
+authors: [
+    "David Malcolm <dmalcolm@redhat.com>"
+]
+homepage: "https://gcc.gnu.org/wiki/JIT"
+license: "GPL-2.0-or-later"
+# Unfortunately, we cannot rely on the OS for selecting the right version
+# of libgccjit-dev, the generic dev package does not seem to exist on some
+# systems. There is libgccjit0, but not libgccjit0-dev (even though there is
+# libgccjit-devel on Fedora, libgccjit on NixOS, libgccjit-dev on Alpine).
+build: [
+  ["sh" "-exc" "gcc --version | grep 12"]
+  ["sh" "-exc" "echo \"#include  \\\"libgccjit.h\\\"\" > test.c"]
+  ["sh" "-exc" "gcc -c $CFLAGS -I/usr/local/include test.c"] {os != "macos"}
+  [
+    "sh"
+    "-exc"
+    "cc -c $CFLAGS -I/opt/homebrew/include -I/opt/local/include -I/usr/local/include test.c"
+  ] {os = "macos"}
+]
+depends: [ "conf-gcc" ]
+# Check that the default version of gcc is not too high.
+# By depending on the specific version of gcc below, we might upgrade
+# the default gcc binary, but we will not downgrade it.
+depexts: [
+  ["libgccjit-12-dev"] {os-family = "debian"}
+  ["libgccjit-12-dev"] {os-family = "ubuntu"}
+  ["gcc-toolset-12-libgccjit-devel"] {os-distribution = "centos"}
+  ["libgccjit-devel.12"] {os-distribution = "fedora"}
+  ["gcc-toolset-12-libgccjit-devel"] {os-distribution = "ol" | os-distribution = "oraclelinux"}
+             | os-family = "debian" | os-family = "ubuntu"
```

Comment:
This line breaks the `depexts` format as also reported on the CI. I'm guessing it is just a copy-paste mistake.
I recommend locally running `opam lint packages/conf-libgccjit/conf-libgccjit.12/opam` to confirm that things are OK (and shamefully admits that I sometimes forget to do so myself... :sweat_smile:)

Reply:
This is an artifact of an accepted suggestion, I don't see it in my repository...